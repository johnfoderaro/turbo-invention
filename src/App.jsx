import React from 'react';
import { hot } from 'react-hot-loader';
import { createGlobalStyle } from 'styled-components';

const GlobalStyle = createGlobalStyle`
  * {
    box-sizing: border-box;
    margin: 0;
    padding: 0;
  }
`;

const App = () => (
  <>
    <GlobalStyle/>
    <h1>
      {'Hello World!'}
    </h1>
  </>
);

export default hot(module)(App);
