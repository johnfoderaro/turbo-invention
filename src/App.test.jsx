import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';

it('should render an H1', () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
