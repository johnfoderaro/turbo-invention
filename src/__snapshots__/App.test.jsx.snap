// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render an H1 1`] = `
<h1>
  Hello World!
</h1>
`;
